error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Diameter: 6.0
TIME:  0.246650218964
CLOCKTIME:  4.320454
	Command being timed: "python /home/cpennycu/networkx/gt.py -f /home/cpennycu/gdata/er/erd-3-100000-16.snap -d 1"
	User time (seconds): 141.66
	System time (seconds): 0.56
	Percent of CPU this job got: 102%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 2:18.18
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 174588
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 57212
	Voluntary context switches: 276
	Involuntary context switches: 6519
	Swaps: 0
	File system inputs: 36816
	File system outputs: 16
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
